publication-file/src/test/java/no/unit/nva/publication/file/upload/DeleteFileHandlerTest.java

```diff
@@ -0,0 +1,59 @@
+package no.unit.nva.publication.file.upload;
+
+import static java.net.HttpURLConnection.HTTP_ACCEPTED;
+import static no.unit.nva.model.testing.PublicationGenerator.randomPublication;
+import static no.unit.nva.model.testing.associatedartifacts.AssociatedArtifactsGenerator.randomOpenFile;
+import static no.unit.nva.publication.PublicationServiceConfig.dtoObjectMapper;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Map;
+import java.util.UUID;
+import no.unit.nva.identifiers.SortableIdentifier;
+import no.unit.nva.publication.service.ResourcesLocalTest;
+import no.unit.nva.publication.service.impl.ResourceService;
+import no.unit.nva.testutils.HandlerRequestBuilder;
+import nva.commons.apigateway.GatewayResponse;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class DeleteFileHandlerTest extends ResourcesLocalTest {
+
+    private static final Context CONTEXT = mock(Context.class);
+    private ResourceService resourceService;
+    private ByteArrayOutputStream output;
+    private DeleteFileHandler handler;
+
+    @BeforeEach
+    void setUp() {
+        super.init();
+        resourceService = getResourceServiceBuilder().build();
+        handler = new DeleteFileHandler(resourceService);
+        output = new ByteArrayOutputStream();
+    }
+
+    @Test
+    void shouldReturnAccepted() throws IOException {
```

Comment:
Can you add a test that does the same for a non-existing file? And potentially a non-existing file on a non-existing publication?

Reply:
Is that the correct/desired behaviour when/if we know that the file/publication doesn't exist? Should it be `404` instead?

Reply:
I will implement functionality and add all the tests once I will start with implementation, it is just a test for skeleton code to satisfy jacoco.